js/form.js

```diff
@@ -0,0 +1,42 @@
+window.addEventListener('DOMContentLoaded', function() {
+    'use strict';
+
+    //Get the form elements defined in your form HTML above
```

Comment:
We should remove these unnecessarily and uninformative comments that came with the original code.